mat64/io_test.go

```diff
@@ -39,3 +49,36 @@ func TestDenseRW(t *testing.T) {
 		}
 	}
 }
+
+func TestVectorRW(t *testing.T) {
+	for i, test := range []*Vector{
+		NewVector(0, []float64{}),
```

Comment:
Also, what do we want to happen if the binary `n` is invalid, such as a negative value or larger than the size of the slice?